### ISSUE TITLE
Doubts in cross_validation_accuracy

### ISSUE THREAD

**Issue by vmanikes** [iit-cs579/main#257]
Hello Professor,
I know these issues have been discussed before but I am unclear how to approach these functions. Firstly in the `cross_validation_accuracy `  in the sklearn's documentation the KFold function is using `class sklearn.cross_validation.KFold(n, n_folds=5)` . How are our parameters `clf, X , Labels` are related to it.

**Comment by aronwc** [comment]
See block [32] in [L11](https://github.com/iit-cs579/main/blob/master/lec/l11/l11.ipynb) for an example use of KFold.

**Comment by davidghiurco** [comment]
@vmanikes the cross_validation_accuracy implementation in Lecture 11 is nearly identical to what we're supposed to implement in a2. this method is not meant to be difficult.
